packaging/installer/methods/macos.md

```diff
@@ -50,10 +50,10 @@ xcode-select --install
 ```
 
 Click **Install** on the Software Update popup window that appears. Then, use the same terminal session to use Homebrew
-to install some of Netdata's prerequisites.
+to install some of Netdata's prerequisites. `cmake` is necessary only if you need to claim the node to Netdata Cloud.
```

Comment:
I would maybe reword it as `cmake is necessary in case you want to use Netdata Cloud functionality` or `you can skip/omit cmake in case you do not want to use Netdata Cloud` but that is subjective. LGTM

Reply:
I prefer not to assume that people know what claiming or netdata cloud are.
```suggestion
to install some of Netdata's prerequisites. `cmake` is necessary only if you need to [claim](https://learn.netdata.cloud/docs/cloud/get-started) the node to [Netdata Cloud](https://netdata.cloud/cloud).
```